core/mps/mock_interface.go

```diff
@@ -37,34 +38,37 @@ func (m *MockPrivateStateManager) EXPECT() *MockPrivateStateManagerMockRecorder
 	return m.recorder
 }
 
-// ResolveForManagedParty mocks base method.
-func (m *MockPrivateStateManager) ResolveForManagedParty(managedParty string) (*PrivateStateMetadata, error) {
+// CheckAt mocks base method.
```

Comment:
did we move the order of functions on this file?

Reply:
It's a generated file with golang mockgen, it might be a newer version